test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,96 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Service Binding with Strimzi Kafka Cluster database without applicationSelector
+        Given Imported Nodejs application "nodejs-empty-app" is running
+        * Strimzi Kafka operator is installed
+        * Kafka cluster "strimzi-kafka-cluster" is running
+        When Service Binding Request is applied to connect the database and the application
+            """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+                name: binding-request-strimzi-kafka-no-app
+            spec:
+                backingServiceSelector:
+                    group: kafka.strimzi.io
+                    kind: Kafka
+                    resourceRef: strimzi-kafka-cluster
+                    version: v1beta1
+                    id: kaf
+                    customEnvVar:
+                      - name: BOOTSTRAP_SERVERS
+                        value: |-
+                            {{- range .kaf.status.listeners -}}
+                                {{- if and (eq .type "tls") (gt (len .addresses) 0) -}}
+                                    {{- with (index .addresses 0) -}}
+                                        {   { .host }}:{{ .port }}
+                                    {{- end -}}
+                                {{- end -}}
+                            {{- end -}}
+
+             """
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "binding-request-strimzi-kafka-no-app" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-strimzi-kafka-no-app" should be changed to "False"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-strimzi-kafka-no-app" should be changed to "EmptyApplicationSelector"
+
+    Scenario: Service Binding with Strimzi Kafka Cluster database
```

Comment:
Could this be renamed to bind an application to Strimzi Kafka Cluster backend service, or something similar?

Reply:
see my comment above - IMHO, this scenario does not focus on the issue being fixed in this PR. It test the same functionality as https://github.com/redhat-developer/service-binding-operator/blob/ba154641ad2f942c13b5d9fee0f175bb21a54582/test/acceptance/features/bindAppToService.feature#L11 but this time using customEnvVar.

Reply:
removing this as part of incoming commit